src/app/core/auth/auth.service.ts

```diff
@@ -156,6 +156,15 @@ export class AuthService {
     return this.store.pipe(select(isAuthenticatedLoaded));
   }
 
+  public setExternalAuthStatus(external: boolean) {
+    this.store.dispatch(new SetAuthCookieStatus(external));
+  }
+
+  public isExternalAuthentication(): Observable<boolean> {
```

Comment:
Same here, can we add TypeDocs.  Something like:  `Returns true if an external authentication system (e.g. Shibboleth) is being used for authentication. Returns false otherwise.`

Reply:
The docs and typos are fixed!